backend/unreferenced_files.csv

```diff
@@ -0,0 +1,4 @@
+69/160/9c26f4d2-92c7-11ee-a053-dd84b614f5d7
```

Comment:
should this file be included ?

Reply:
nope 😆 But i just realised this file comes from running the script tests. so adding code to remove the file as part of this issue.